README.md

```diff
@@ -95,54 +95,79 @@ A stream is a set of log lines with a unique set of labels.
 
 #### Method `client.instantQuery(query, limit)`
 
+This function is a shortcut for `client.instantQueryAt(query, limit, time.Now())` where `time.Now()` is the current nanosecond.
+
+#### Method `client.instantQueryAt(query, limit, instant)`
+
 Execute an instant query ([GET /loki/api/v1/query](https://grafana.com/docs/loki/latest/api/#get-lokiapiv1query)).
 
-| argument | type | description | default |
-| -------- | ---- | ----------- | ------- |
-| query | string | The LogQL query to perform. | - |
-| limit | integer | Maxiumum number of entries to return. | - |
+| argument | type           | description                           | default |
+|----------|----------------|---------------------------------------|---------|
+| query    | string         | The LogQL query to perform.           | -       |
+| limit    | integer        | Maxiumum number of entries to return. | -       |
+| instant  | 64-bit integer | Nanosecond at which to execute query. | -       |
```

Comment:
I think we can simply write `integer` as type, because there is only a [Number](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#number-type) type, which is a double-precision floating point (64 bit) number. (Well, there is also [bigint](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#bigint_type) ...)

```suggestion
| instant  | integer | Nanosecond at which to execute query. | -       |
```

:smirk: 
![2022-06-08_09-18](https://user-images.githubusercontent.com/281260/172555802-c2025297-b25e-4c47-bff1-3bde8ce033f3.png)

Reply:
Good point, done :+1: